src/signalhandler.cc

```diff
@@ -145,7 +145,7 @@ void DumpTimeInfo() {
 void DumpStackFrameInfo(const char* prefix, void* pc) {
   // Get the symbol name.
   const char *symbol = "(unknown)";
-  char symbolized[1024];  // Big enough for a sane symbol.
+  char symbolized[1024] = {};  // Big enough for a sane symbol.
```

Comment:
Perhaps this was not the case on Linux, but on Windows, the `Symbolize()` only filled the buffer as far as the symbol took. So you'd run into this kind of problem:
```
[uninitializedbufferofrandomdata]
[main()ializedbufferofrandomdata]
[someLongSymbol()ferofrandomdata]
[short()gSymbol()ferofrandomdata]
```
Since the buffer was not being initialized, but the same data location was being reused and only partially filled out, previous longer symbols left gunk in buffer with new shorter symbols.

I solved this by ensuring all elements were initialized to zero each time `DumpStackFrameInfo` was called. In retrospect, if I ensure a the terminating null is added manually, this becomes unnecessary.